### PR TITLE
Updated

### PR DESCRIPTION
**Pull request by sodasoba1**
Added more terms to a few menu items.

### CODE REVIEW

File: docs/extras/tools.md [hunk 1]
```diff
@@ -39,4 +39,9 @@ Here are some tools that might be useful for theming and other types of modding.
     Kuriimu2 is a general purpose game translation project manager and toolkit for authors of fan translations and game mods. This tool also works for the Switch's menus. [Here's an example](https://github.com/bandithedoge/switch-pl) of a fan-translation in Polish.
 -   [Game Icon Customizer](https://github.com/hotshotz79/NX-Game-Icon-Customizer)  
     **Supported filetype:** N/A  
-    An application to simplify the method of changing game icons in Atmosphere.
+    An application to simplify the method of changing game icons in Atmosphere via your pc.
+-   [IconGrabber](https://github.com/Slluxx/IconGrabber)
+
+    **Supported filetype:** N/A
+
+    A homebrew method of changing game icons in Atmosphere via your switch. (steamgriddb api key required).
```
Comment (Migushthe2nd) [inline docs/extras/tools.md:47]: Please fix the spacing here

**Review by Migushthe2nd — CHANGES_REQUESTED** [review]
Good additions in general. 
One comment: 
- Please make sure every page has a proper 'return button' (make sure to use the octicon icons and header style).

**Comment by Migushthe2nd** [comment]
Good additions in general, but make sure every page has a return button at the top (and bottom if applicable) and that they are properly styled (with the octicon icons and header style).

**Comment by sodasoba1** [comment]
Excellent stuff I'll look into it further, I'm new to mkdocs so it's just a learning curve, I'll do my best to keep the style required, if you can supply a blank template that I can easily adapt that page for further additions

**Pull request closed without merging**